about.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bring to dream mentorship</title>
+
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <!--Stylesheet-->
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="header-container">
+      <div class="nav-banner">
+        <div class="banner-icons">
+          <ul class="social-icons">
+            <li>
+              <a href="#facebook"></a><i class="fa-brands fa-facebook-f"></i>
+            </li>
+            <li><a href="#twitter"></a><i class="fa-brands fa-twitter"></i></li>
+            <li><a href="#english"></a>English</li>
+            <li><a href="#myPage"></a>My Page</li>
+            <li><a href="#logout"></a>Logout</li>
+          </ul>
+        </div>
+      </div>
+      <div class="header-nav">
+        <div class="navbar">
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+          <div class="logo">
+            <a href="index.html"
+              ><img src="./assets/images/logo.PNG" alt="merctech mentorship"
+            /></a>
+          </div>
+          <nav class="nav-menu">
+            <ul class="menu-links">
+              <li>
+                <a href="index.html" class="a-links">Home</a>
+              </li>
+              <li>
+                <a href="#program" class="a-links">Program</a>
+              </li>
+              <li>
+                <a href="#join" class="a-links">Join</a>
+              </li>
+              <li>
+                <a href="#sponsor" class="a-links">Sponsor</a>
+              </li>
+              <li>
+                <a href="#news" class="a-links">News</a>
+              </li>
+              <li class="campaign">
+                <a href="#campaign" class="a-links">campaign</a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+    <div class="header-contents-about">
+      <div class="container-herooo">
+        <div class="landing-text-about">
+          <div>
+            <img
+              src="./assets/images/theme1.PNG"
+              class="hero-img"
+              alt="theme"
+            />
```

Comment:
Hey 👋🏽 seems like you just used an image here 🖼️  but for this section, it should follow the guidelines below:
It contains a top section with a column that includes a small line of text, the page title, a block of text, a medium line of text in bold letters, and a smaller line of text. 🤓